drivers/filesystems/fs_rec/fs_rec.c

```diff
@@ -430,13 +430,13 @@ DriverEntry(IN PDRIVER_OBJECT DriverObject,
                              0);
     if (NT_SUCCESS(Status)) DeviceCount++;
 
-    /* Register EXT2 */
+    /* Register EXT */
     Status = FsRecRegisterFs(DriverObject,
                              NULL,
                              NULL,
-                             L"\\Ext2fs",
-                             L"\\FileSystem\\Ext2Recognizer",
-                             FS_TYPE_EXT2,
+                             L"\\Extfs",
```

Comment:
Be careful: Shouldn't this device name match the corresponding device  name that will be created by the filesystem driver itself?
(See ext2/inc/ext2fs.h DEVICE_NAME, and CDROM_NAME as well. Similarly to what is done also for the FAT filesystem.)

Reply:
it doesn't have to, in the sense it's not a hard requirements, it'll work without being aligned.
It's already not aligned, Ext2fs vs Ext2Fsd in the driver.  
But it can be re-aligned if you prefer.
what has to be aligned is [this](https://github.com/reactos/reactos/blob/master/drivers/filesystems/fs_rec/ext2.c#L98) and the service name in the [driver](https://github.com/reactos/reactos/blob/master/drivers/filesystems/ext2/ext2fs_reg.inf)

Reply:
If the nettles don't need perfect alignment, then your current proposal is fine.

Reply:
@Vince1171 could you consider adding the ext cdrom recognition, though?

Reply:
Ext CD's Recognizer added